HLTrigger/Configuration/scripts/hltGetConfiguration

```diff
@@ -243,8 +242,19 @@ parser.add_argument('--help',
 # parse command line arguments and options
 config = parser.parse_args(namespace = options.HLTProcessOptions())
 
-cmdArgs = sys.argv
-cmdArgs[0] = os.path.basename(sys.argv[0])
+# do not include db-proxy options in 1st-line comment
+cmdArgs, skipNext = [], False
+for cmdArg in sys.argv:
+  if skipNext:
+    skipNext = False
+    continue
+  if cmdArg.startswith('--dbproxy'):
+    if cmdArg.startswith('--dbproxyh') or cmdArg.startswith('--dbproxyp'):
+      skipNext = '=' not in cmdArg
+    continue
+  cmdArgs += [cmdArg]
+cmdArgs[0] = os.path.basename(cmdArgs[0])
+
```

Comment:
This avoids having `--dbproxy` in the 1st line of the output of `hltGetConfiguration`. The db-proxy option should arguably be "untracked", so to speak. Plus, without this we would have spurious diffs in the first line of TSG menus dumped in the release (depending on how `getHLT.sh` was run).